payloads/management/v3.3/configuration-api/responses/agents/listAgents.json

```diff
@@ -1,16 +1,24 @@
 [
   {
+    "avatar_path": "cdn.livechatinc.com/avatars/1.png",
     "id": "smith@example.com",
     "job_title": "Support Hero",
+    "login_status": "not accepting chats",
+    "name": "Smith",
+    "role": "owner",
     "max_chats_count": 5,
     "summaries": [
       "daily_summary",
       "weekly_summary"
     ]
   },
   {
+    "avatar_path": "cdn.livechatinc.com/avatars/2.png",
     "id": "adam@example.com",
     "job_title": "Support Hero (Newbie)",
+    "login_status": "accepting chats",
+    "name": "Adam",
```

Comment:
Jones or Brown pls, not some Adam 😉

Reply:
sure, I named them such as they had names like that in the ids 😅